### ISSUE TITLE
Windows salt minion can't always autoload the custom salt module 

### ISSUE THREAD

**Issue by WangWenchao** [saltstack/salt#30099]
As I have about half of server is Windows server and the others is Linux Server . when I  have a common module for both linux and windows OS，  but after excute the command 

`salt '*' saltutil.sync_modules`

Linux minion  almost can autoload the salt minion 
but some of my Windows minions can autload the module , some cant autoload ,  everytime I have to restart  the salt minion manually or use other batch tools to restart the salt minion on Window Server , its too bad experience

**Comment by cachedout** [comment]
Interesting. So, it sounds like for whatever reason we aren't fully syncing to the Windows minions. A few questions:

* Could you please provide the output of `salt --versions-report` on one Windows minions and on the salt master?

* Please provide debug logs from the salt daemon running on the Windows minion.

* Could you please clarify what you mean by 'autoload'? Is the module not available when you try to run functions from it on via the `salt` command from the master?

**Comment by WangWenchao** [comment]
@cachedout  thanks,  the salt master version report 
```
[root@server ~]# salt --versions-report
Salt Version:
           Salt: 2015.8.3

Dependency Versions:
         Jinja2: 2.2.1
       M2Crypto: 0.20.2
           Mako: 1.0.3
         PyYAML: 3.10
          PyZMQ: 14.3.1
         Python: 2.6.6 (r266:84292, Jul 23 2015, 15:22:56)
           RAET: Not Installed
        Tornado: 4.2.1
            ZMQ: 3.2.5
           cffi: Not Installed
       cherrypy: 3.2.2
       dateutil: 1.4.1
          gitdb: Not Installed
      gitpython: Not Installed
          ioflo: Not Installed
        libnacl: Not Installed
   msgpack-pure: Not Installed
 msgpack-python: 0.4.6
   mysql-python: 1.2.5
      pycparser: Not Installed
       pycrypto: 2.6.1
         pygit2: Not Installed
   python-gnupg: Not Installed
          smmap: Not Installed
        timelib: Not Installed

System Versions:
           dist: centos 6.4 Final
        machine: x86_64
        release: 3.13.6
         system: CentOS 6.4 Final
```

 here  'autoload' mean  my custom  salt  module synced  by saltutil.sync_modules  is not available when I excute  from the master. After I restart the Windows salt-minion service   manually , it works normally

**Comment by twangboy** [comment]
@WangWenchao Could you try  setting `multiprocessing=False` in your minion config and see if the problem persists?

**Comment by WangWenchao** [comment]
@twangboy  thanks for your reply .
My salt minion config setting ```multiprocessing: False ``` is already set

**Comment by twangboy** [comment]
@WangWenchao Hmmm. It should be set to `True` by default on 2015.8.3. Maybe try setting it to True.

**Comment by WangWenchao** [comment]
has fixed on version 2015.8.7

**Comment by twangboy** [comment]
@WangWenchao So, can this be closed?
